scm/src/main/java/com/sandboni/core/scm/resolvers/ChangeScopeFromCliDiffCall.java

```diff
@@ -0,0 +1,157 @@
+package com.sandboni.core.scm.resolvers;
+
+import com.sandboni.core.scm.cli.GitDiffRunner;
+import com.sandboni.core.scm.exception.SourceControlException;
+import com.sandboni.core.scm.exception.SourceControlRuntimeException;
+import com.sandboni.core.scm.proxy.SourceControlFilter;
+import com.sandboni.core.scm.revision.RevisionScope;
+import com.sandboni.core.scm.scope.*;
+import com.sandboni.core.scm.utils.RawUtil;
+import com.sandboni.core.scm.utils.ThrowingConsumer;
+import io.reflectoring.diffparser.api.model.Diff;
+import io.reflectoring.diffparser.api.model.Hunk;
+import io.reflectoring.diffparser.api.model.Line;
+import io.reflectoring.diffparser.api.model.Range;
+import org.eclipse.jgit.diff.RawText;
+import org.eclipse.jgit.lib.ObjectId;
+import org.eclipse.jgit.lib.Repository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+public class ChangeScopeFromCliDiffCall extends ChangeScopeResolver implements ChangeScopeResolverInterface {
+
+    private final Repository repository;
```

Comment:
Make the super.repository protected and use it here.

Reply:
done